.idea/modules.xml

```diff
@@ -3,7 +3,9 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
+      <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
```

Comment:
With this and the previous comment, I usually don't mess with the IDE config files, these changes have been automatic and I'd prefer let android studio do it's own thing